src/compiler/scala/tools/nsc/typechecker/ContextErrors.scala

```diff
@@ -1041,7 +1041,9 @@ trait ContextErrors {
     self: Inferencer =>
 
     private def applyErrorMsg(tree: Tree, msg: String, argtpes: List[Type], pt: Type) = {
-      def asParams(xs: List[Any]) = xs.mkString("(", ", ", ")")
+      def asParams(xs: List[Any]) =
+        if (xs.isEmpty && tree.symbol.isConstructor) "empty args"
```

Comment:
"empty args" is a bit casual, maybe "an empty argument list"?

Reply:
I'll let it say "empty args" only on casual Fridays.